clients/venice-client/src/main/java/com/linkedin/venice/fastclient/meta/VeniceClientBasedMetadata.java

```diff
@@ -160,7 +160,7 @@ public void start() {
         Thread.currentThread().interrupt();
       }
     }
-    scheduler.scheduleAtFixedRate(this::refresh, 0, refreshIntervalInSeconds, TimeUnit.SECONDS);
+    scheduler.scheduleAtFixedRate(this::refresh, refreshIntervalInSeconds, refreshIntervalInSeconds, TimeUnit.SECONDS);
```

Comment:
Thanks for finding and fixing this. Can you please add a comment here to add some randomness to this to distribute the refresh over a minute rather than all sending request together? Or wondering whether these servers starting at different time will automatically provide the randomness.

Reply:
Yeah. My assumption is it is inherently random due to different deployment times. Given a large enough fleet size, randomness won't really help any way